generator/transit_generator.cpp

```diff
@@ -213,12 +181,13 @@ void DeserializerFromJson::operator()(FeatureIdentifiers & id, char const * name
         ("Cann't convert osm id string:", osmIdStr, "to a number."));
   osm::Id const osmId(osmIdNum);
   auto const it = m_osmIdToFeatureIds.find(osmId);
-  CHECK(it != m_osmIdToFeatureIds.cend(), ("Osm id:", osmId, "(encoded", osmId.EncodedId(),
-                                           ") is not found in osm id to feature ids mapping."));
-  CHECK_EQUAL(it->second.size(), 1,
-              ("Osm id:", osmId, "(encoded", osmId.EncodedId(),
-               ") from transit graph doesn't present by a single feature in mwm."));
-  id.SetFeatureId(it->second[0]);
+  if (it != m_osmIdToFeatureIds.cend())
+  {
+    CHECK_EQUAL(it->second.size(), 1,
+                ("Osm id:", osmId, "(encoded", osmId.EncodedId(),
+                 ") from transit graph is not represented by any feature in the mwm."));
```

Comment:
What if second.size() is 2? Or 3? Is it possible? If so, the error message is not correct.

Reply:
I rewrote the error message:
```
    CHECK_EQUAL(it->second.size(), 1, ("Osm id:", osmId, "(encoded", osmId.EncodedId(),
                 ") from transit graph is correspond to several features or is not represented by "
                 "any feature in the mwm."));
```

Reply:
Note that it would be helpful to log it->second too.

Reply:
I wrote this way:
```
    CHECK_EQUAL(it->second.size(), 1, ("Osm id:", osmId, "(encoded", osmId.EncodedId(),
                 ") from transit graph is correspond to", it->second.size()" features."
                 "But osm id should be represented be one feature."));
```